WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentsTransactionAlertsProvider.swift

```diff
@@ -33,6 +33,7 @@ struct CardPresentPaymentsTransactionAlertsProvider: CardReaderTransactionAlerts
     }
 
     func error(error: any Error,
+               receiptState: CardReaderTransactionFailureAlertReceiptState,
```

Comment:
Just for clarification: The changes in this class is to "adapt the adapter" right? We're passing the receipt state as well across to the POS domain so we can reuse this too?

Reply:
POS could reuse it but it's not necessary. Like POS didn't reuse email callbacks from success provider.

We concluded that untangling receipts from the payment flow would make sense. However, it would require refactoring success and email alert providers in some way as well, so they would support adding receipt options but wouldn't be coupled with it.